RELEASE.md

```diff
@@ -1,3 +1,10 @@
+# Release order
+1. Check the `RELEASE-ORDER.md` file.
+2. If there are multiple modules affected by this version, release should be created from top to bottom (the ones with less dependencies are released first, so after their release, the dependent ones can be released). So, with the new release of the `capture-sdk`, `bank-sdk` should be released as well.
+3. `capture-sdk:default-network` version is always bumped when the `capture-sdk` is updated.
+4. `core-api-library`, `bank-api-library` and `health-api-library` are not released separately, they are released with the `capture-sdk`, `bank-sdk` and `health-sdk` releases.
```

Comment:
"`core-api-library` and `capture-sdk:default-network` don't get their own release notes, but they still need to be released separately. For `core-api-library` we share release notes through `bank-api-library` and `healht-api-library`. For `capture-sdk:default-network` we share the release notes through `capture-sdk`. "

Reply:
I just the desired changes. Thanks :-)